__tests__/app.test.js

```diff
@@ -0,0 +1,47 @@
+const request = require('supertest')
+const app = require("../app")
+const db = require("../db/connection")
+const seed = require('../db/seeds/seed');
+const testData = require('../db/data/test-data/index');
+
+beforeEach(() => seed(testData));
+
```

Comment:
Make sure you add afterAll here too to end the connection with the database after testing - this will avoid getting open handles errors.